views/agreement.hbs

```diff
@@ -114,20 +114,84 @@ input[type="submit"] {
+     copyrights and patents, as set forth in this Contributor and Workstream Participant Agreement
+     (“<dfn id="agreement">Agreement</dfn>”) and the written
+     <a href="https://whatwg.org/policies">WHATWG Policies</a>. <a href="#individual">Individuals</a>,
+     other than those invited by the
+     <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>, who
+     <a href="https://whatwg.org/ipr-policy#2101-work-in-the-field-of-web-technologies">work in the field of web technologies</a>
+     as an employee, contractor, or agent of another person or legal entity
+     must have this <a href="#agreement">Agreement</a> executed by an authorized representative of
+     that <a href="#entity">Entity</a>. (To make
+     <a href="https://whatwg.org/ipr-policy#21-contribution">Contributions</a>, an
+     <a href="#entity">Entity</a> must be a
+     <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participant</a>.)
 
   <p>Terms not defined here are defined in the <a href="https://whatwg.org/policies">WHATWG Policies</a>.
  </div>
 
  <p>“<dfn id="entity">Entity</dfn>” means a legal entity that makes <a href="https://whatwg.org/ipr-policy#21-contribution">Contributions</a> through its employees, agents, contractors, and other representatives; "<dfn id="affiliate">Affiliate</dfn>" means any legal entity that directly or indirectly <a href="#control">Controls</a>, is <a href="#control">Controlled</a> by, or is under common <a href="#control">Control</a> with an entity. "<dfn id="control">Control</dfn>" means the power to determine the course of action of another entity (for example by ownership of more than 50% of the voting stock or equity, authority to appoint a majority of the board or equivalent body, or by virtue of its voting or other rights).
 
- <p>“<dfn id="individual">Individual</dfn>” means a person who does not <a href="https://whatwg.org/ipr-policy#2101-work-in-the-field-of-web-technologies">work in the field of web technologies</a> as an employee, contractor, or agent of another person or legal entity, and who thus makes <a href="https://whatwg.org/ipr-policy#21-contribution">Contributions</a> in their personal capacity only.
-
+ <p>“<dfn id="individual">Individual</dfn>” means one of the following:
+
+ <ul>
+  <li>
+    a person who does not
+    <a href="https://whatwg.org/ipr-policy#2101-work-in-the-field-of-web-technologies">work in the field of web technologies</a>
+    as an employee, contractor, or agent of another person or legal entity, and who thus makes
+    <a href="https://whatwg.org/ipr-policy#21-contribution">Contributions</a> in their personal capacity
+    only, or
+  <li>
+    a person who has been invited by the
+    <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> to participate in one or more WHATWG
+    <a href="https://whatwg.org/workstream-policy#workstream">Workstreams</a> for a specified term
+    ("Invited Individual's Term")
```

Comment:
We should use curly quotes here to be consistent, but since it's not a definition I'm not actually sure if that would be consistent and maybe it should be a reference instead?

Reply:
Yes, I actually think this is supposed to be a definition. It's used again in the Termination section with no quotes.

Reply:
Pushed a change with this as a dfn. Happy to try something else if this doesn't look right.

Reply:
Looks good. (I didn't review the formatting pass.)